components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -1015,7 +1015,7 @@ open class FxaAccountManager(
 
     @VisibleForTesting
     open fun createAccount(config: ServerConfig): OAuthAccount {
-        return FirefoxAccount(config)
+        return FirefoxAccount(config, null, crashReporter)
```

Comment:
I'm a bit confused by why the `PersistCallback` is being passed as null here - is there maybe some other creator of a `FirefoxAccount` which is passed a non-null `PersistCallback` but will now be getting a null `crashReporter`?

Reply:
as mentioned on slack, this should be fine, just a bad API for FirefoxAccount. It will register a persist callback internally regardless of what you pass in here.